python/orca/example/learn/bigdl/attention/transformer.py

```diff
@@ -20,7 +20,7 @@
 from tensorflow.python.keras.datasets import imdb
 from tensorflow.python.keras.preprocessing import sequence
 
-from bigdl.dllib.keras.models import Model
+from bigdl.dllib.keras.model import Model
```

Comment:
This is not correct; it should be either `from bigdl.dllib.keras import Model` or `from bigdl.dllib.keras.models import Model`